.github/workflows/build.yaml

```
@@ -49,91 +49,82 @@ jobs:
-#          helm: "3.8.0"
-#      - name: Login to GitHub Container Registry
-#        uses: docker/login-action@v1
-#        with:
-#          registry: ghcr.io
-#          username: ${{ github.repository_owner }}
-#          password: ${{ secrets.GITHUB_TOKEN }}
-#          logout: false
-#      - name: push
-#        run: |
-#          sha_ver=sha-$(git rev-parse --short HEAD)
-#          tag_ver=${{ github.event.release.tag_version }}
-#          explicit_version=$(printf '%s\n' "${app_ver}-${sha_ver}" | sed -e 's/[\/.]/\\./g')
-#          chart_ver="${tag_ver:-${explicit_version}}"
-#          image_tag="${tag_ver:-${sha_ver}}"
-#          cd charts
-#          for d in */ ; do
-#              if [ -f "$d/Chart.yaml" ]; then
-#                echo "Setting Chart.yaml version to ${chart_ver}..."
-#                sed -i "s/^version\: .*/version\: ${chart_ver}/" $d/Chart.yaml
-#                sed -i "s/^appVersion\: .*/appVersion\: ${app_ver}/" $d/Chart.yaml
-#                sed -i "s/^  tag\: main/  tag\: ${image_tag}/" $d/values.yaml
-#                echo "Packaging Helm chart $d..."
-#                helm package $d --app-version="${app_ver}"
-#                echo "Packaged Helm chart $d with chart version [${chart_ver}] and appVersion to [${app_ver}]"
-#                echo *.tgz
-#                helm push *.tgz oci://ghcr.io/${{ github.repository_owner }}/charts
-#                rm *.tgz
-#              fi
-#          done
-#        env:
-#          app_ver: ${{ needs.app.outputs.VERSION || '0.0.0' }}
+          setup-tools: helm
+          helm: "3.8.0"
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+          logout: false
+      - name: push
+        run: |
+          sha_ver=sha-$(git rev-parse --short HEAD)
+          tag_ver=${{ github.event.release.tag_version }}
+          explicit_version=$(printf '%s\n' "${app_ver}-${sha_ver}" | sed -e 's/[\/.]/\\./g')
+          chart_ver="${tag_ver:-${explicit_version}}"
+          image_tag="${tag_ver:-${sha_ver}}"
+          cd charts
+          for d in */ ; do
+              if [ -f "$d/Chart.yaml" ]; then
+                echo "Setting Chart.yaml version to ${chart_ver}..."
+                sed -i "s/^version\: .*/version\: ${chart_ver}/" $d/Chart.yaml
+                sed -i "s/^appVersion\: .*/appVersion\: ${app_ver}/" $d/Chart.yaml
+                sed -i "s/^  tag\: main/  tag\: ${image_tag}/" $d/values.yaml
+                echo "Packaging Helm chart $d..."
+                helm package $d --app-version="${app_ver}"
+                echo "Packaged Helm chart $d with chart version [${chart_ver}] and appVersion to [${app_ver}]"
+                echo *.tgz
+                helm push *.tgz oci://ghcr.io/${{ github.repository_owner }}/charts
+                rm *.tgz
+              fi
+          done
```

You don't need the sed commands:
- `helm package` has options to set version and appVerision
- tag defaults to appVersion

```suggestion
          cd charts/abacus
          helm package $d --app-version="${app_ver}" --version="${chart_ver}"
          echo "Packaged Helm chart $d with chart version [${chart_ver}] and appVersion to [${app_ver}]"
          echo *.tgz
          helm push *.tgz oci://ghcr.io/${{ github.repository_owner }}/charts
          rm *.tgz
```